automation_tools/satellite6/upgrade/tests/utils.py

```diff
@@ -0,0 +1,152 @@
+            time.sleep(5)
+        hostname = '{0}dockerclient{1}'.format(count, client_os)
+        container_id = run(
+            'docker run -d -h {0} -e "SATHOST={1}" '
+            '-e "AK={2}" upgrade:{3}'.format(
+                hostname, satellite_hostname, ak, client_os))
+        result[hostname] = container_id
+    return result
+
+
+def get_id(entity, name, org_id):
+    result = hammer_info_via_name(entity, name, org_id)
+    return get_attribute_value(result, name, 'id')
+
+
+def get_latest_repo_version(cv_name):
+    cv_version_data = hammer(
+        'content-view version list --content-view {} '
+        '--organization-id 1'.format(cv_name))
+    latest_cv_ver = sorted([float(data['name'].split(
+        '{} '.format(cv_name))[1]) for data in cv_version_data]).pop()
+    cv_ver_id = get_attribute_value(cv_version_data, '{0} {1}'.format(
+        cv_name, latest_cv_ver), 'id')
+    return cv_ver_id
+
+
+def create_vm():
+    usecase_instance = rhev_template + 'usecase-testing'
+    delete_rhevm_instance(usecase_instance, timeout=5)
+    create_rhevm_instance(usecase_instance,
+                          rhev_template,
+                          'SAT-QE',
+                          'SAT-QE',
+                          'SAT-QE')
+
+
+def dockerize(ak_name):
+    docker_vm = os.environ.get('DOCKER_VM')
+    # Check if the VM containing docker images is up, else turn on
+    # change to os args
+    rhevm_client = get_rhevm_client()
+    instance_name = 'sat6-docker-upgrade'
+    template_name = 'sat6-docker-upgrade-template'
+    vm = rhevm_client.vms.get(name=instance_name)
+    if not vm:
+        logger.info('Docker VM for generating Content Host is not created.'
+                    'Creating it, please wait..')
+        create_rhevm_instance(instance_name, template_name)
+        execute(manage_daemon, 'restart', 'docker', host=docker_vm)
+    elif vm.get_status().get_state() == 'down':
+        logger.info('Docker VM for generating Content Host is not up. '
+                    'Turning on, please wait ....')
+        rhevm_client.vms.get(name=instance_name).start()
+        wait_till_rhev_instance_status(instance_name, 'up', 5)
+        execute(manage_daemon, 'restart', 'docker', host=docker_vm)
+    rhevm_client.disconnect()
+    time.sleep(5)
+    logger.info('Generating 1 client on RHEL6 and RHEL7 on Docker. '
+                'Please wait .....')
+    # First delete if any containers running
+    execute(
+        remove_all_docker_containers, only_running=False, host=docker_vm)
+    time.sleep(5)
+    # Generate Clients on RHEL 7
+    time.sleep(30)
+    clients = execute(
+        generate_satellite_docker_clients_on_rhevm_upgrade,
+        'rhel7',
+        1,
+        ak_name,
+        host=docker_vm,
+    )[docker_vm]
+    return clients
+
+
+def docker_return_code_last(container_id):
+    result = docker_execute_command(container_id, 'echo $?', quiet=True)
+    return result
+
+
+def get_satellite_host():
```

Comment:
The `get` accepts a second parameter which is the default value if the key is not defined. This is equivalent code if one env var will available:

```python
return os.environ.get(
    'RHEV_SAT_HOST',
    os.environ.get('SATELLITE_HOSTNAME')
) 
```

If both might be available and but the first can be empty string then the code is equivalent:

```python
return os.environ.get('RHEV_SAT_HOST') or environ.get('SATELLITE_HOSTNAME')
```